app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.append("JavaScript")
+        self.skills.append("HTML")
+        self.skills.append("CSS")
```

Comment:
That is a bad practice, see the checklist: 
https://github.com/mate-academy/py-software-engineers/blob/master/checklist.md#%D1%81heck-your-code-against-the-following-points